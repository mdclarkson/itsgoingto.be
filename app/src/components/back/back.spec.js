import React from 'react'
import { shallow } from 'enzyme'
import Back from './back'

let wrapper

describe('(Component) Back', () => {
  beforeEach(() => {
    wrapper = shallow(<Back />)
  })

  describe('(Action) onClick', () => {
    browserHistory.push = jest.fn()

    it('should call submit', () => {
      wrapper.find('a').simulate('click')
      expect(browserHistory.push).toBeCalledWith('/')
    })
  })

  describe('(Render)', () => {
    it('matches snapshot', () => {
      expect(wrapper).toMatchSnapshot()
    })
  })
})
