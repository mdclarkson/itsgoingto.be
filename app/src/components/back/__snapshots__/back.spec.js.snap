// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`(Component) Back (Render) matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="container back-container"
  >
    <div
      class="back-button"
    >
      <a
        href="/"
      >
        <i
          class="fa fa-arrow-left"
        />
        New Poll
      </a>
    </div>
  </div>
</DocumentFragment>
`;
